actionview/lib/action_view/helpers/translation_helper.rb

```diff
@@ -85,8 +85,11 @@ def translate(key, options = {})
             end
           end
           translation = I18n.translate(scope_key_by_partial(key), html_safe_options.merge(raise: i18n_raise))
-
-          translation.respond_to?(:html_safe) ? translation.html_safe : translation
+          if translation.is_a?(Array)
```

Comment:
Can `I18n.translate` return enumerables other than just Array? E.g. would `respond_to?(:map)` be clearer/better?